pkgs/top-level/all-packages.nix

```diff
@@ -4359,6 +4359,10 @@ in
 
   which = callPackage ../tools/system/which { };
 
+  chase = callPackage ../tools/system/chase {
+    inherit boehmgc;
```

Comment:
`callPackage` is supposed to fill in all missing parameter from the enclosing scope; this explicit `inherit` ought to be unnecessary here.